### ISSUE TITLE
Two websites, two domains, one server

### ISSUE THREAD

**Issue by 4thearth** [Eugeny/ajenti-v#141]
Ubuntu 14.04
Ajenti-V

So I have two websites that I've created and set up within /srv/ lets call one '/something/' and the other one '/somthingelse/'. Now I own the domain 'something.com' and 'somethingelse.com' but I can only get the first one to show up correctly. Both are running off of port 80 and domains are entered repsectivley but I dont know what wrong.

/something/ is set to 'something.com' domain and to host '@'  on port 80
/somethingelse/ is set to 'somethingelse.com' domain to host '@' on port 80

**Comment by Teepareep** [comment]
paste your vhost files /etc/nginx/conf.d/* here

**Comment by 4thearth** [comment]
'''
server {
    listen *:80;
    
    
    server_name postichproducts.com;

    access_log /var/log/nginx/postich.access.log;
    error_log /var/log/nginx/postich.error.log;

    root /srv/postich;
    index index.html index.htm index.php;

    location ~ [^/]\.php(/|$) {
        
        fastcgi_index index.php;
        include fcgi.conf;
        fastcgi_pass unix:/var/run/ajenti-v-php-fcgi-postich-php-fcgi-0.sock;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;

    }

}
'''

and 

'''
server {
    listen *:4322;
    
    
    

    access_log /var/log/nginx/vics.access.log;
    error_log /var/log/nginx/vics.error.log;

    root /srv/vics;
    index index.html index.htm index.php;

    

    
    
    location ~ [^/]\.php(/|$) {
        
        
        
        fastcgi_index index.php;
        include fcgi.conf;
        fastcgi_pass unix:/var/run/ajenti-v-php-fcgi-vics-php-fcgi-0.sock;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;

    }

}

'''

Right now it is configured so that my main site runs through port 80 and works best with GoDaddy's DNS settings and the second site is running on port 4322 and using forward masking which I am not happy about at all

**Comment by Teepareep** [comment]
where's the server_name directive in the second vhost?

**Comment by 4thearth** [comment]
server {
    listen *:4322;
    
    
    server_name vicspizzaoncoffee.com;

    access_log /var/log/nginx/vics.access.log;
    error_log /var/log/nginx/vics.error.log;

    root /srv/vics;
    index index.html index.htm index.php;

    

    
    
    location ~ [^/]\.php(/|$) {
        
        
        
        fastcgi_index index.php;
        include fcgi.conf;
        fastcgi_pass unix:/var/run/ajenti-v-php-fcgi-vics-php-fcgi-0.sock;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;

    }

}

**Comment by Teepareep** [comment]
If you set both to port 80, what happens when you visit the second domain?

**Comment by 4thearth** [comment]
First domain works but the second one wont load

**Comment by Teepareep** [comment]
Do you get an error? Does it just hang? Specificity goes a long way.

**Comment by 4thearth** [comment]
No its like its not even finding it. Doesn't even attempt to load anything
because I don't think it's configured right. I have both hosts set to *
along with both GoDaddy domains hosts set *. Is it possible to change hosts
to specific names (like can I use any host name I want?) so I could set
server1 to host "postich" and point godaddys DNS to postich on port 80?

I dont know how well formed this question is I am still trying to learn
this as I go.

**Comment by 4thearth** [comment]
Another update:
If I set a subdomain using GoDaddys directions:
https://support.godaddy.com/help/article/4080/managing-a-domain-names-subdomains

and set the second site to the host "vics" as well as added the subdomain
host "vics" to godaddy.


    server {
    listen *:80;


    server_name postichproducts.com;

    access_log /var/log/nginx/postich.access.log;
    error_log /var/log/nginx/postich.error.log;

    root /srv/postich;
    index index.html index.htm index.php;





    location ~ [^/]\.php(/|$) {



        fastcgi_index index.php;
        include fcgi.conf;
        fastcgi_pass
unix:/var/run/ajenti-v-php-fcgi-postich-php-fcgi-0.sock;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;

     }

     }

    server {
    listen vics:80;


    server_name vicspizzaoncoffee.com;

    access_log /var/log/nginx/vics.access.log;
    error_log /var/log/nginx/vics.error.log;

    root /srv/vics;
    index index.html index.htm index.php;





    location ~ [^/]\.php(/|$) {



        fastcgi_index index.php;
        include fcgi.conf;
        fastcgi_pass unix:/var/run/ajenti-v-php-fcgi-vics-php-fcgi-0.sock;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;

    }

    }

and now when I load vics.postichproducts.com still only the postichproducts
main site loads

**Comment by 4thearth** [comment]
I also now have this on my gnix config
nginx: [emerg] host not found in "vics:80" of the "listen" directive in
/etc/nginx/conf.d/vics.conf:7 nginx: configuration file
/etc/nginx/nginx.conf test failed

**Comment by brianjking** [comment]
@4thearth Did you ever get this sorted out?

**Comment by 4thearth** [comment]
@brianking yes but this was a while ago I think I had to add lines to the hosts file of Ubuntu then set the hosts in ajenti to BOTH *:80 and (hostname):80 on both websites
